### PR TITLE
Updated Covid Policy Doc

### LINKED ISSUE #401: Update /covid to reflect new masking guidelines
Changes are necessary to https://acm.cs.ucla.edu/covid to reflect UCLA's new masking policy

Changes requested

- Change Last major update: Monday, January 17th 2022.
- Add a bullet point to Version history with date Monday, January 17th 2022. and the following copy:
updated to reflect [Updated masking, isolation and quarantine protocols for UCLA](https://covid-19.ucla.edu/updated-masking-isolation-and-quarantine-protocols-for-ucla/)
- Download ACM at UCLA COVID Policies Winter 2022 as pdf and update Link to PDF button to point to this new PDF

### PR DESCRIPTION
**Pull request by ArvinDing**
resolves  #401

### CODE REVIEW

File: pages/covid.js [hunk 1]
```diff
@@ -29,10 +29,11 @@ function COVID() {
           Our plan is approved by <a href="https://sole.ucla.edu/" target="_blank" rel="noopener noreferrer">UCLA SOLE</a> and is a superset of <a href="https://covid-19.ucla.edu/ucla-return-to-campus/" target="_blank" rel="noopener noreferrer">UCLA&apos;s return to campus</a> and <a href="http://ph.lacounty.gov/acd/ncorona2019/EducationToolkit/IHE/" target="_blank" rel="noopener noreferrer">LA County Guidelines</a>.
         </p>
         <p>
-          Last major update: <b>Friday, January 7th 2021</b>.
+          Last major update: <b>Friday, January 17th 2022</b>.
```
Comment (advaithg) [inline pages/covid.js:32]: I think this changes to Monday as well?

**Review by advaithg — CHANGES_REQUESTED** [review]
Other than the day change, looks good!

File: pages/covid.js (hunk 1 above)
Comment (ArvinDing) [reply pages/covid.js]: whoops

Comment (ArvinDing) [reply pages/covid.js]: just changed that

Comment (advaithg) [reply pages/covid.js]: no worries!

**Review by advaithg — APPROVED** [review]

**Pull request merged**